index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <script src="https://kit.fontawesome.com/9dfa895fe8.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <div class="content">
+    <div class="navbar">
+      <div class="logo">
+        <h2>My Logo</h2>
+      </div>
+      <div class="icon">
+        <img src="images/Union.jpg" alt="Union">
+      </div>
+    </div>
+    <div class="section">
```

Comment:
- [ ] Let's change the `div` tag to the `<section>` tag for the introduction section to improve the structure and semantics.